src/index.html

```diff
@@ -6,8 +6,47 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
     <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <img
+        class="card__img"
+        src="./images/imac.jpeg"
+        alt="photo of apple monitor"
+      >
+
+      <div class="card__description">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </div>
+
+      <div class="card__product-code">Product code: 195434</div>
+
+      <div class="stars">
+        <ul class="stars__list">
+          <li class="stars__list-item"></li>
+          <li class="stars__list-item"></li>
+          <li class="stars__list-item"></li>
+          <li class="stars__list-item"></li>
+          <li class="stars__list-item"></li>
+        </ul>
```

Comment:
this is only stars BEM block
```suggestion
        <ul class="card__stars stars">
          <li class="stars__item"></li>
          <li class="stars__item"></li>
          <li class="stars__item"></li>
          <li class="stars__item"></li>
          <li class="stars__item"></li>
        </ul>
```